core/core.cc

```diff
@@ -317,18 +317,17 @@ void TaskGraph::execute_point(long timestep, long point,
     std::vector<std::pair<long, long> > deps = dependencies(dset, point);
     for (auto span : deps) {
       for (long dep = span.first; dep <= span.second; dep++) {
-        if (last_offset <= dep && dep < last_offset + last_width) {
+      	if (last_offset <= dep && dep < last_offset + last_width) {
```

Comment:
Can we put this indentation back to avoid having this diff? (And next 2 occurrences.) Thanks.